sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImpl.java

```diff
@@ -259,9 +317,13 @@ public SerializerAdapter getSerializerAdapter() {
 
         @Post("/sentiment")
         @ExpectedResponses({200})
+        @UnexpectedResponseExceptionType(
+                value = ErrorResponseException.class,
+                code = {400, 500})
```

Comment:
Bit confused on why this is generating as it is the same type as the default

Reply:
If i remember correctly, it is new behavior from autogene? @srnagar

Reply:
The swagger defines specific error responses for these HTTP status codes. Unfortunately, they use the same type that catches the generic error. Ideally, the swagger should define a more specific error type for 400 and 500 or not define it at all if it's the same error for all other status codes.